### DISCUSSION TITLE
How to give claim blocks each hour

### DISCUSSION THREAD

**Discussion by godgutten** [GriefPrevention/GriefPrevention#1174] [Q&A] [answered]
Hey how do i make it so my players get 500 claim blocks each hour? And how can I make my VIPs get 2x claim blocks each hour?

**Comment by Jikoo** [comment, marked as answer]
The answer to your first question is on the wiki in [this section](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#claim-limits). The second is available via addon, you'll need [GPAccruals](https://github.com/TechFortress/GriefPrevention/wiki/Addons#gpaccruals).
